packages/serverless/src/domain/use-case/schema.ts

```diff
@@ -166,7 +166,7 @@ export async function getSchemaDefinition(
     (fieldName) =>
       ({
         name: schema[fieldName].name,
-        kind: schema[fieldName].king,
+        kind: schema[fieldName].kind,
```

Comment:
this problem fixed in the latest code, you can pull it

Reply:
done